app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -146,7 +145,7 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         pager = root.findViewById(R.id.pager);
         pager.setAdapter(new AudioPlayerPagerAdapter(this));
         // Required for getChildAt(int) in ViewPagerBottomSheetBehavior to return the correct page
-        pager.setOffscreenPageLimit((int) NUM_CONTENT_FRAGMENTS);
+        pager.setOffscreenPageLimit(NUM_CONTENT_FRAGMENTS);
```

Comment:
This change is unrelated to the checkbox, please revert. Note that the cast is there for a reason, see https://github.com/AntennaPod/AntennaPod/commit/260d7ee9446258508a2b9145267bfe858a0e9540

Reply:
Done. Not sure how that got changed, definitely wasn't intentional.